<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotTemplateListWidget</class>
 <widget class="QWidget" name="PlotTemplateListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>24</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QStackedWidget" name="stwdtList">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="templateListEditPage">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lExistTemplateList">
          <property name="text">
           <string>Plot template list</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="tvExistTemplateList"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="templateListViewPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="LibraryPlotTemplateWidget" name="wdtTemplateList" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>24</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lAddTemplate">
           <property name="text">
            <string>Add plot template</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbNextStep">
           <property name="text">
            <string>Next step</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbAddExistTemplate">
           <property name="text">
            <string>Add template</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbCancel">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbControlMenu">
           <property name="text">
            <string>Template control</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="stwdtTemplate">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="templateViewPage">
          <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
           <item>
            <widget class="QScrollArea" name="scrollArea_2">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>65</width>
                <height>16</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="PlotTemplateViewWidget" name="wdtDetails" native="true"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="templateEditPage">
          <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>706</width>
                <height>473</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="PlotTemplateEditWidget" name="wdtEditTemplate" native="true"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="EmptyPage"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotTemplateViewWidget</class>
   <extends>QWidget</extends>
   <header>ui/widgets/library/plot_template_view_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlotTemplateEditWidget</class>
   <extends>QWidget</extends>
   <header>ui/widgets/library/plot_template_edit_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LibraryPlotTemplateWidget</class>
   <extends>QWidget</extends>
   <header>ui/widgets/library/library_plot_template_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
